contracts/protocol/facets/DisputeResolverHandlerFacet.sol

```diff
@@ -188,40 +207,192 @@ contract DisputeResolverHandlerFacet is IBosonAccountEvents, ProtocolBase {
         // Check that msg.sender is the admin address for this dispute resolver
```

Comment:
"Check that caller is the admin..."

Reply:
done

Reply:
It doesn't look like this change actually got made.